tutor/src/components/add-edit-question/ux.js

```diff
@@ -0,0 +1,240 @@
+import { action, observable, computed } from 'vendor';
+import { filter, some, find, forEach, pickBy } from 'lodash';
+import { TAG_BLOOMS, TAG_DOKS } from './form/tags/constants';
+
+export default class AddEditQuestionUX {
+
+  // local
+  @observable showAddEditForm = false;
+  @observable showAddEditTermsOfUse = false;
+
+  @observable isEmpty = {
+    selectedChapter: false,
+    selectedChapterSection: false,
+    questionText: false,
+  }
+
+  // props
+  //@observable question;
+  @observable book;
+  @observable course;
+  // chapter/section ids
+  @observable pageIds;
+
+  /** form */
+  // chapter/sections
+  @observable selectedChapter;
+  @observable selectedChapterSection;
+  // question
+  @observable questionText;
+  @observable answerKeyText;
+  @observable isTwoStep;
+  @observable options = [];
+  @observable detailedSolution;
```

Comment:
@camliu89 @nathanstitt I think you can send answerKeyText and detailedSolution as the same field

Reply:
@jivey yes. I realized that as I was working on my other branch.